sunshine/process.cpp

```diff
@@ -182,7 +184,27 @@ void proc_t::terminate() {
       std::abort();
     }
   }
+ auto &proc = _apps[app_id];
 
+  _undo_begin = std::begin(proc.disconnect_cmds);
+  _undo_it    = _undo_begin;
+
+for(; _undo_it != std::end(proc.disconnect_cmds); ++_undo_it) {
+    auto &cmd = _undo_it->do_cmd;
+
+    BOOST_LOG(info) << "Executing: ["sv << cmd << ']';
+    auto ret = exe(cmd, _env, _pipe, ec);
+
+    if(ec) {
+      BOOST_LOG(error) << "Couldn't run ["sv << cmd << "]: System: "sv << ec.message();
+      return 0;
```

Comment:
`return false;`

Reply:
also, should there be a `return` at all within a for loop?

Reply:
Removed returns.